mlpm_ex01_linear_regression/1_LinRegTut_STUDENT.ipynb

```diff
@@ -385,15 +385,14 @@
    ]
   },
   {
-   "cell_type": "raw",
+   "cell_type": "markdown",
    "metadata": {
     "pycharm": {}
    },
    "source": [
-    "No. The covariance of x and sin(x) will be 0,\n",
-    "given a intervall thats a multiple of 2*pi.\n",
-    "But the relation between x and y can be described as y=sin(x), thus \n",
-    "they are not independet of each other."
+    "No. \n",
+    "\n",
+    "For example, the covariance of x and sin(x) is 0.000029, given an intervall that's a multiple of 2*pi. But the relation between x and y can be described as $ y=sin(x) $, which means that they are not independent of each other."
```

Comment:
die Kovarianz von x und sin(x) für das intervall ist 0, wenn die Zahl der werte gegen unendlich geht.
Ich würde das nochmal umformulieren... irgendwie

Reply:
For example, the covariance of x and sin(x) is 0, given $x\in\R$ and $0\leg x \leq 2\pi$ that's a multiple of 2*pi.